tests/validation_suite/automation_framework/utilities/workflow.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import logging
+import time
+import json
+
+from avalon_client_sdk.http_client.http_jrpc_client import HttpJrpcClient
+from error_code.error_status import WorkOrderStatus
+logger = logging.getLogger(__name__)
+
+
+def submit_request(uri_client, input_json_str1, output_json_file_name):
+    """ Function to submit request to enclave,
```

Comment:
submit request to enclave -> submit request to Avalon listener

Reply:
Done